templates/ceph/shared/_latest-news-ceph.html

```diff
@@ -0,0 +1,44 @@
+<section class="p-section--deep">
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+    <div class="p-section--shallow">
+      <h2>
+        <a href="{{ title_link }}">Latest news from&nbsp;our&nbsp;blog&nbsp;&rsaquo;</a>
+      </h2>
+    </div>
+  </div>
+  <div class="row p-section--shallow" id="ceph-install-latest"></div>
+
+  <template style="display:none" id="ceph-install-template">
```

Comment:
```suggestion
  <template id="ceph-install-template">
```
Might be wrong here, but I think `template`'a are hidden from the page flow by default

Reply:
You're right, updated!